addons/account/models/uom_uom.py

```diff
@@ -19,27 +19,13 @@ def _get_unece_code(self):
         https://unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_rev3_Annex2e.pdf"""
         mapping = {
             'uom.product_uom_unit': 'C62',
-            'uom.product_uom_dozen': 'DZN',
+            'uom.product_uom_pack_6': 'HD',
             'uom.product_uom_kgm': 'KGM',
             'uom.product_uom_gram': 'GRM',
             'uom.product_uom_day': 'DAY',
             'uom.product_uom_hour': 'HUR',
```

Comment:
I recommend holding off on the uom cleaning for a follow up task since I think whoever suggested their removal didn't realize how many lookup values it's going to remove. I already mentioned the GS1 codes to you, but it looks like these UNECE code removals here aren't even possible for the customers to easily add new records to replace them